index.html

```diff
@@ -23,7 +23,7 @@
           </li>
         </ul>
       </nav>
-      <div id="date">5pm eastern time</div>
+      <div id="time">Febuary 9th 2022, 1:43pm</div>
```

Comment:
- [ ] Kindly remove this div. The current date and time under the navbar should be provided by Luxon library and not hardcoded. 👍